includes/Loading/LoadCombination.js

```diff
@@ -1,19 +1,259 @@
-function LoadCombination(no,
-                         design_situation,
-                         items,
-                         comment,
-                         params)
-{
-    var items = typeof items !== 'undefined' ? items : [];
+/*
+Not tested:
+- Consider initial state from
+- Calculate critical load (in default option is disabled)
+- Creep caused by pernament load (in default option is disabled)
+- Consider construction state (in default option is disabled)
+Not implemented:
+- Assignment for CO2
+*/
 
-    var combination = engine.create_load_combination(no);
-    combination.design_situation = design_situation;
+/**
+ * Creates load combination
+ * @param {Number}  no      Load combination index, can be undefined
+ * @param {String}  name    Name, can be undefined
+ * @param {String}  comment Comment, can be undefined
+ * @param {Object}  paramns Additonal parameters, can be undefined
+ * @returns Created load combination
+ */
+function LoadCombination (no,
```

Comment:
Function have to remain the same as in current GM - function LoadCombination(no,
                         design_situation,
                         items,
                         comment,
                         params)